app/views/users/new.html.erb

```diff
@@ -0,0 +1,48 @@
+<h1>Create new User</h1>
+<% if false %> 
+<form action="/users" method="POST" accept-charset="UTF-8">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <label for="username">Username</label>
+  <input type="text" name="user[username]" id="username"><br><br>
```

Comment:
- [ ] this form field should be also commented out. [We only leave](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_for.png) uncommented the form_for part.